README.md

```diff
@@ -1,23 +1,21 @@
-# Sending Data to Tanzu Observability by Wavefront
+# Sending Data to Our Service
 
-If you use OpenTelemetry, you can configure the application to send traces or metrics to Tanzu Observability using the Wavefront Proxy or the OpenTelemetry Collector.
+If you use OpenTelemetry, you can configure the application to send traces or metrics to VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront) using the Wavefront proxy or the OpenTelemetry Collector.
 
-## Directly Send Data Using the Wavefront Proxy - (Recommended)
+Send trace data or metrics data to our service using only the Wavefront proxy:
```

Comment:
Not sure if WF proxy is still called WF proxy.

Reply:
We still call it Wavefront proxy :)